app/controllers/events/view/sessions/list.js

```diff
@@ -60,6 +60,18 @@ export default class extends Controller.extend(EmberTableControllerMixin) {
           addRating    : this.addRating.bind(this)
         }
       },
+      {
+        name          : this.l10n.t('Average Rating'),
+        width         : 90,
+        valuePath     : 'averageRating',
+        cellComponent : 'ui-table/cell/events/view/sessions/cell-rating-details'
+      },
+      {
+        name          : this.l10n.t('No. of ratings'),
+        width         : 90,
+        valuePath     : 'feedbacks.length',
+        cellComponent : 'ui-table/cell/events/view/sessions/cell-rating-details'
+      },
```

Comment:
This should be one column

Reply:
@iamareebjamal sir by one column, do you mean like this,

![Screenshot_2021-01-30 Session Sessions brio Events Open Event(1)](https://user-images.githubusercontent.com/65965202/106333574-f473e680-62ae-11eb-8ba2-bfc62170bea6.png)

Reply:
Rating and Average Rating/No. of rating should all be in one column

Reply:
ok, i am making the changes